### PR TITLE
CI: run jenkins as batman

### PR DESCRIPTION
**Pull request by jnewb1**
I suspect the unable-to-delete-workspace might be because it's creating files as root that can't be deleted

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -81,10 +81,12 @@ def pcStage(String stageName, Closure body) {
 
     checkout scm
 
-    def dockerArgs = '--user=root -v /tmp/comma_download_cache:/tmp/comma_download_cache -v /tmp/scons_cache:/tmp/scons_cache';
+    def dockerArgs = '--user=batman -v /tmp/comma_download_cache:/tmp/comma_download_cache -v /tmp/scons_cache:/tmp/scons_cache';
     docker.build("openpilot-base:build-${env.GIT_COMMIT}", "-f Dockerfile.openpilot_base .").inside(dockerArgs) {
       timeout(time: 20, unit: 'MINUTES') {
         try {
+          sh "sudo chown -R batman:batman /tmp/scons_cache"
```
Comment (jnewb1) [inline Jenkinsfile:88]: similar to the devcontainer, this can be removed eventually when there are no jenkins jobs running as root

**Comment by adeebshihadeh** [comment]
Have you tested this works by removing the cleanup in the finally block?

**Comment by jnewb1** [comment]
> Have you tested this works by removing the cleanup in the finally block?

seems to work fine

workspace created here: https://jenkins.comma.life/blue/organizations/jenkins/openpilot/detail/jenkins-as-batman/9/pipeline/28, not cleaned up because I removed it
cleaned up in the next replay: https://jenkins.comma.life/blue/organizations/jenkins/openpilot/detail/jenkins-as-batman/10/pipeline/28#step-80-log-3

File: Jenkinsfile (hunk 1 above)
Comment (adeebshihadeh) [reply Jenkinsfile:88]: make sure to leave a comment for stuff like this

**Review by adeebshihadeh — APPROVED** [review]
LGTM, just add a comment about the chown migration stuff and remove the cleanup

**Pull request merged**